### ISSUE TITLE
tiltfile-result (alpha) executes with error

### ISSUE THREAD

**Issue by Yshayy** [tilt-dev/tilt#4981]
### Expected Behavior
Using tilt alpha tiltfile-result -f staging.Tiltfile should provide the result of tilt execution

### Current Behavior
```
Loading Tiltfile at: /workspace/staging.Tiltfile
Successfully loaded Tiltfile (1.065042027s)
Error: encoding JSON: Could not print: json: unsupported type: apiset.ObjectSet
```

### Steps to Reproduce

1. Using tilt alpha tiltfile-result
2. This is reproducible even with empty TiltFile

Some notes:
- It works when adding the "-b" flag
- It worked properly with v0.22.2

#### `tilt doctor` Output

```
Tilt: v0.22.9, built 2021-09-17
System: linux-amd64
---
Docker
- Host: [default]
- Server Version: 20.10.6
- API Version: 1.41
- Builder: 2
---
Kubernetes
- Env: unknown
- Context: eks-staging-env
- Cluster Name: kubernetes
- Namespace: default
- Container Runtime: docker
- Version: v1.21.2-eks-0389ca3
- Cluster Local Registry: none
---
```

#### About Your Use Case
We explore the possibility of using the output of tiltfile-result for checking diff before running tilt ci.

**Comment by nicks** [comment]
ahh! thank you for reporting!!

**Comment by nicks** [comment]
we just released v0.22.10, which should fix this problem
